teaching-platfrom/controller.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask
+from flask import request
+import db_handler
+
+
+
+app = Flask(__name__)
+
+@app.route("/add", methods=["POST"])
+def add_user():
+    request_data = request.get_json()
+    username, password = request_data.get("username"), request_data.get("password")
+
+    if not username or not password: return "Incorrect json body", 400
```

Comment:
You might consider some validation of provided username and password. Typical library for password validation is [password-strength](https://pypi.org/project/password-strength/).